src/common/components/header/header.scss

```diff
@@ -30,7 +30,7 @@
     gap: 8px;
   }
 
-  .navbar--items a {
+  &__items a {
```

Comment:
This notation is totally fine, but it can also be moved within the first items class:
`&__items {
    ...styles;
    a { 
       ...styles
    }
}`